qiita_pet/support_files/doc/source/tutorials/prepare-templates.rst

```diff
@@ -0,0 +1,13 @@
+Templates
+=========
+
+The terms *Sample template* and *Prep template* have been depreciated. We now
+call these the *Sample information file* and *Preparation information files*. 
+
+We will now redirected you to :doc:`prepare-information-files`.
+
+.. raw:: html
+	<meta http-equiv="refresh" content="10;url=https://qiita.ucsd.edu/static/doc/html/tutorials/prepare-information-files.html">
```

Comment:
Terrible misuse of HTML. I'm looking for a better way to redirect.
(Perhaps I could submit a blank .html file with a redirect, instead of trying to generate it with Sphynx.)